chia/wallet/transaction_sorting.py

```diff
@@ -0,0 +1,14 @@
+import enum
+
+
+class SortKey(str, enum.Enum):
```

Comment:
it's not clear to me why this is a class. it doesn't use any of the features of classes. These are just constants

Reply:
I'm open to other suggestions but the idea was just for it to be an enum like https://github.com/Chia-Network/chia-blockchain/blob/main/chia/types/condition_opcodes.py